superdesk/utils.py

```diff
@@ -35,6 +35,32 @@ class SortOrder(Enum):
     desc = 2
 
 
+@unique
```

Comment:
Can we think of any use cases where we would want non-unique values?
But there is a point in this, and we can as well remove this restriction from the base class, yes (though it's trivial to lift the restriction later, if needed).

Personally I'm indifferent here, both options seem fine...